src/plugins/kibana/public/settings/styles/main.less

```diff
@@ -211,3 +211,18 @@ kbn-settings-indices {
     }
   }
 }
+
+.wizard-step-title {
```

Comment:
Yeah, these will probably get moved out when I try to do some refactoring when creating additional wizards.